### PR TITLE
Expose walltime kwarg on CLI

### LINKED ISSUE #772: Expose dask-worker walltime on CLI
Currently one cannot tune the walltime used for the dask workers in jobqueue mode dynamically. 
We should fix this so that time can be extended if needed.

### PR DESCRIPTION
**Pull request by hmacdope**
## Description
Provide a brief description of the PR's purpose here.
Fixes #772 

## Developers certificate of origin
- [X] I certify that this contribution is covered by the MIT license as defined in our [LICENSE](https://github.com/choderalab/asapdiscovery/blob/main/LICENSE) and adheres to the [**Developer Certificate of Origin**](https://developercertificate.org/).

### CODE REVIEW

**Comment by hmacdope** [comment]
@kaminow @apayne97 @JenkeScheen @jthorton can I grab a review here?

File: asapdiscovery-data/asapdiscovery/data/dask_utils.py [hunk 1]
```diff
@@ -286,7 +286,12 @@ class LilacGPUDaskCluster(LilacDaskCluster):
     cores = 1
 
     @classmethod
-    def from_gpu(cls, gpu: GPU = GPU.GTX1080TI, silence_logs: int = logging.DEBUG):
+    def from_gpu(
+        cls,
+        gpu: GPU = GPU.GTX1080TI,
+        silence_logs: int = logging.INFO,
+        walltime: str = "72h",
```
Comment (kaminow) [inline asapdiscovery-data/asapdiscovery/data/dask_utils.py:293]: should this be passed into the constructor below?

**Review by kaminow — APPROVED** [review]
one quick fix, but other than that looks good!

File: asapdiscovery-data/asapdiscovery/data/dask_utils.py (hunk 1 above)
Comment (hmacdope) [reply asapdiscovery-data/asapdiscovery/data/dask_utils.py:293]: Indeed it should :eagle: :eyes:

**Pull request merged**